doc/api-hidden.rst

```diff
@@ -613,6 +613,7 @@
    CFTimeIndex.fillna
    CFTimeIndex.floor
    CFTimeIndex.format
+   CFTimeIndex.freq
```

Comment:
I dont see `freq` in API reference: https://xray--4597.org.readthedocs.build/en/4597/generated/xarray.CFTimeIndex.html. do I have to run some extra script for sphinx? @spencerkclark

Reply:
it's in the `Attributes` section. `api-hidden.rst` is used to generate API pages for attributes/methods, `autosummary` will always create a entry for the object, but you can't click on the name if the API page is not generated.

It would probably be good to move `CFTimeIndex.freq` to the section below (in `api-hidden.rst`, the one that starts with `CFTimeIndex.T`), but that's just to make it easier for a reader, it doesn't matter for sphinx.